packages/backend/src/open_payments/receiver/service.ts

```diff
@@ -45,8 +57,30 @@ export async function createReceiverService(
   }
 
   return {
-    get: (url) => getReceiver(deps, url)
+    get: (url) => getReceiver(deps, url),
+    create: (url) => createReceiver(deps, url)
+  }
+}
+
+async function createReceiver(
+  deps: ServiceDependencies,
+  args: CreateReceiverArgs
+): Promise<Receiver> {
+  const remoteIncomingPayment = await deps.remoteIncomingPaymentService.create(
+    args
+  )
+
+  const receiver = Receiver.fromIncomingPayment(remoteIncomingPayment)
```

Comment:
As the comment above, do we want to keep the validation in `Receiver.fromIncomingPayment` ? https://github.com/interledger/rafiki/blob/07461933be2f4e90c350d99396bbdcf09944401a/packages/backend/src/open_payments/receiver/model.ts#L23-L48

Would this be better as separate functions, and have the caller of `Receiver.fromIncomingPayment` decide if a Receiver is "legit" or not?